app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpCreationViewModel.kt

```diff
@@ -111,6 +111,7 @@ class MeetUpCreationViewModel @Inject constructor(
         _tags.value = emptySet()
         _participantsId.value = listOf(profileService.getLoggedInUserID()!!)
         _criterionAge.value = Pair(13, 66)
```

Comment:
We could move those into the integers.xml file (I know not part of your work but I notice it now